### DISCUSSION TITLE
Keep getting `OutOfMemoryError`

### DISCUSSION THREAD

**Discussion by ethankongee** [TencentARC/PhotoMaker#114] [Q&A] [answered]
I'm using `g4dn.2xlarge` on AWS with 14.58 GB total memory, but I keep getting `OutOfMemoryError`. How much memory do we need for this model? Which GPU / service are you guys using?

**Comment by Paper99** [comment]
Did you try to switch `dtype` to `torch.float16`?

**Reply by ethankongee** [reply, marked as answer]
Yeah still got the same error.

But I found [this article](https://huggingface.co/blog/simple_sdxl_optimizations#:~:text=So%20if%20you%20run%20SDXL,memory%20and%20take%2072.2%20seconds!&text=This%20isn't%20very%20practical,generating%20more%20than%204%20images.) and added `pipe.enable_vae_slicing()`.. it's working now.

**Reply by Paper99** [reply]
Thanks!
